app/decorators/new_statement_decorator.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+# Decorator with merges statements with up-to-date position held data
+class NewStatementDecorator < SimpleDelegator
+  attr_accessor :comparison, :type
+
+  def initialize(statement, comparison)
+    @comparison = comparison
+    super(statement)
+  end
+
+  def statement_uuid
+    if exact?
+      comparison.exact_matches.first
+    elsif conflict?
+      comparison.conflicts.first
+    else
+      comparison.partial_matches.first
+    end
+  end
```

Comment:
In my testing I saw some statements being flagged as "There were no 'position held' (P39) statements on Wikidata that match the actioned suggestion", but actually the statement_uuid was in the SPARQL query output. I _think_ that's because `comparison.exact_matches`, `comparison.conflicts` and `comparison.partial_matches` can all match more than one statement. So I think this method should possibly be called `statement_uuids` and return _all_ relevant statements, rather than just `.first`, for the cases where we match more than one existing statement.

Does that make sense?

Reply:
Do you have a example I could look at?

Reply:
This should never be nil. Do you have a example I could look at?

Reply:
This looks to be a case we haven't address in the membership comparison gem yet where an existing statement was started during a term. Will fix there. Don;t think this should hold up deploying this tho.